### PR TITLE
feat: adds default search sort by publish date

### PR DESCRIPTION
**Pull request by redreceipt**
also fixes algolia indexing and universal links 😬

Screen Shot 2021-07-22 at 2 44 13 PM

### CODE REVIEW

File: packages/apollos-data-connector-algolia-search/src/data-source.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ if (ApollosConfig.ALGOLIA.APPLICATION_ID && ApollosConfig.ALGOLIA.API_KEY) {
   INDEX.setSettings(
     ApollosConfig.ALGOLIA.CONFIGURATION || {
       searchableAttributes: ['title', 'unordered(summary)'],
+      customRanking: ['desc(publishDate)'],
```
Comment (vinnyjth) [inline packages/apollos-data-connector-algolia-search/src/data-source.js:25]: shouldn't we sort by relevancy first, then publishDate?

File: packages/apollos-server-core/src/index.js [hunk 2]
```diff
@@ -196,9 +196,11 @@ export const createContextGetter = (serverConfig) => (data) => {
 
 export const createMiddleware = (data) => ({ app, context, dataSources }) => {
   const middlewares = compact(
-    mapValues(
-      { ...builtInData, ...data },
-      safeGetWithWarning('serverMiddleware')
+    Object.values(
```
Comment (vinnyjth) [inline packages/apollos-server-core/src/index.js:199]: Good catch!

File: packages/apollos-data-connector-algolia-search/src/data-source.js (hunk 1 above)
Comment (redreceipt) [reply packages/apollos-data-connector-algolia-search/src/data-source.js:25]: yea it does, custom ranking comes after the text input in the event of a tie

Comment (vinnyjth) [reply packages/apollos-data-connector-algolia-search/src/data-source.js:25]: 💥

**Review by vinnyjth — APPROVED** [review]

File: packages/apollos-data-connector-algolia-search/src/data-source.js (hunk 1 above)
Comment (redreceipt) [reply packages/apollos-data-connector-algolia-search/src/data-source.js:25]: still don't understand the difference between using `ranking` and `customRanking` for sorting though. both seemed to have the same effect when testing

**Pull request merged**